Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchSelector.cc

```diff
@@ -27,25 +27,25 @@
 
 //#define EDM_ML_DEBUG
 
-namespace alCaIsolatedBunchSelector {
+namespace AlCaIsolatedBunch {
```

Comment:
why are you doing this?
https://cms-sw.github.io/cms_coding_rules.html
2.7 says the original was good

Reply:
Or at least the first character of the original was good :)